mkdocs.yml

```diff
@@ -37,6 +42,7 @@ plugins:
       raise_error_excludes:
         429: ["https://github.com/ponylang", "https://web.archive.org/*"]
         404: ["https://github.com/ponylang"]
+        504: ["https://web.archive.org/*"]
```

Comment:
this appears unrelated.

Reply:
That's temporary due to https://app.netlify.com/sites/pony-tutorial/deploys/66608a3635679b0008f1cff3#L152